### DISCUSSION TITLE
VictorOps Notifier doesn't populate `custom_fields`

### DISCUSSION THREAD

**Discussion by ottaviohartman** [grafana/grafana#38268] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
We should add tags in Grafana alerts to the [`custom_fields`](https://help.victorops.com/knowledge-base/incident-fields-glossary/) of a VictorOps notification.

The PagerDuty notifier [has this functionality](https://github.com/grafana/grafana/blob/93c39ca18cf1a4bb6a57574ff5436e4328198be9/pkg/services/alerting/notifiers/pagerduty.go#L185) so the implementation would be similar

**Why is this needed**:
Adding tags to alerts is a great way to organize them and populate them with contextual information. This would also help improve the VictorOps notifier's features to match PagerDuty more closely.

I can submit a PR soon to fix this.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
